service/templates/admin.html

```diff
@@ -1502,10 +1502,6 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<label>Local storage size</label>
 										<input type="number" ng-model="Settings.amboy.local_storage_size">
 									</md-input-container>
-									<md-input-container class="control" style="width:45%;">
-										<md-checkbox ng-model="Settings.amboy.remote_require_priority">Remote Require
-											Priority</md-checkbox>
-									</md-input-container>
```

Comment:
Since this option is being removed, it should also be removed from the admin settings data model.

Reply:
haha I was looking for `remote_require_priority`  and couldn't find it so I assumed we must have already removed it. But after you asked I decided to double check and found `require_remote_priority`.
Now it's removed for real.

Reply:
one second, pushing in a minute